pkgs/dartpad_ui/README.md

```diff
@@ -13,3 +13,22 @@ flutter run -d chrome --web-browser-flag "--disable-web-security"
 We pass the `--disable-web-security` flag to Chrome as we're not able to
 configure the `flutter run` web server to pass CORS headers for
 `AssetManifest.json`, `FontManifest.json`, and other resources.
+
+## How to connect to local backend
+
+1. In the directory `pkgs/dart_services` run:
+
+    ```
+    dart tool/grind.dart build-project-templates
```

Comment:
Here and a few places below, you can remove the 4 space indent for the code fence ('```'). A code block either needs the surrounding triple back tick, or to be indented by 4 spaces, but not both.

Reply:
It is not for code block, it is for aligning indentation with numeric items:

![Screenshot 2025-03-03 at 3 30 54 PM](https://github.com/user-attachments/assets/62cd6c99-0df4-4c3d-a14d-374786a02a2f)

Reply:
Ah, gotcha.